plugin/bind/README.md

```diff
@@ -11,7 +11,7 @@ another IP instead.
 
 If several addresses are provided, a listener will be open on each of the IP provided.
 
-Each address has to be an IP of one of the interfaces of the host.
+Each address has to be an IP or name of one of the interfaces of the host. If the given argument is an interface name, and that interface has serveral IP addresses, CoreDNS will listen on all of the interface IP addresses.
```

Comment:
To address @SuperQ 's concerns, please document that this binds to the IPs on that interface at the time the time of startup or reload (reload will happen with a SIGHUP or if the config file changes).

Also verify that this is true; i.e., that reload results in binding to the new addresses, if the addresses have changed since the initial load.

Reply:
Thanks. I've added these to README. I will also test and verify this behavior so please don't merge this until I verified this.

Reply:
@johnbelamaric  BTW I also verified this (both on SIGHUB and with reload plugin).